src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -208,7 +208,7 @@ def _get_hosts(self, format='plain'):
             table.align = 'l'
             table.left_padding_width = 0
             table.right_padding_width = 1
-            for node in completion.result:
+            for node in sorted(completion.result, key=lambda h: h.name):
                 table.add_row((node.name, node.addr, ' '.join(node.labels)))
             output = table.get_string()
```

Comment:
Works for me.

prettytable also supports sorting:
https://github.com/jazzband/prettytable#sorting-your-table-by-a-field